vmdb/app/controllers/application_controller/buttons.rb

```diff
@@ -93,11 +93,11 @@ def automate_button_field_changed
 #     @resolve[:saved_buttons] = Array.new
 #     aset = CustomButtonSet.find_by_name("#{@resolve[:target_class].to_s.downcase}_custom")
 #     if aset
-#       aset.members.sort{|a,b| a.button_id <=> b.button_id }.each do |as|
+#       aset.members.sort_by(&:button_id).each do |as|
```

Comment:
I don't want to be in the business of deleting these in this PR...can definitely do separately

I just want this PR to be pure find/replace.